### PR TITLE
Fix docker system info model class.

### PR DESCRIPTION
**Pull request by garagatyi**
Also add comment with missing fields. 
I haven't added these fields because it is not the target of this PR.

@skabashnyuk @riuvshin @mmorhun Please review

### CODE REVIEW

File: plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/json/SystemInfo.java [hunk 1]
```diff
@@ -303,92 +343,150 @@ public void setName(String name) {
...
+               && Objects.equals(operatingSystem, that.operatingSystem)
+               && Objects.equals(systemTime, that.systemTime)
+               && Arrays.equals(systemStatus, that.systemStatus);
     }
 
-    public void setNoProxy(String noProxy) {
-        this.noProxy = noProxy;
+    @Override
+    public int hashCode() {
+        int hash = 7;
```
Comment (mmorhun) [inline plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/json/SystemInfo.java]: Why not `Objects.hash()` ?

Comment (benoitf) [inline plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/json/SystemInfo.java]: yep looks like half of the class is about equals and hashcode

Comment (garagatyi) [inline plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/json/SystemInfo.java]: Changed to less verbose version of equals and hashcode

**Comment by mmorhun** [comment]
+1

**Comment by AndrienkoAleksandr** [comment]
+1

**Pull request merged**